modules/sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service.adoc

```diff
@@ -41,82 +37,58 @@ EOD
 The output verifies that the `ServiceBinding` CR is created to project the binding data into the sample application.
 +
 .Example output
+[source,terminal]
 ----
-servicebinding.binding.operators.coreos.com/spring-petclinic-rest created
+servicebinding.binding.operators.coreos.com/spring-petclinic created
 ----
 
-. To verify that the binding is successful, check the status conditions of the binding resource:
+. Verify that the request for service binding is successful:
 +
+[source,terminal]
 ----
-$ oc get servicebindings spring-petclinic-rest -n my-postgresql -o jsonpath-as-json='{.status.conditions}'
+$ oc get servicebindings -n my-petclinic
 ----
 +
 .Example output
+[source,terminal]
 ----
-[
-  [
-    {
-      "lastTransitionTime": "2021-09-06T13:42:28Z",
-      "message": "",
-      "reason": "DataCollected",
-      "status": "True",
-      "type": "CollectionReady",
-    },
-    {
-      "lastTransitionTime": "2021-09-06T13:42:28Z",
-      "message": "",
-      "reason": "ApplicationUpdated",
-      "status": "True",
-      "type": "InjectionReady",
-    },
-    {
-      "lastTransitionTime": "2021-09-06T13:42:28Z",
-      "message": "",
-      "reason": "ApplicationsBound",
-      "status": "True",
-      "type": "Ready",
-    },
-  ],
-]
+NAME                          READY   REASON              AGE
+spring-petclinic-postgresql   True    ApplicationsBound   47m
 ----
 +
-By default, the values from the binding data of the database service are projected as files into the workload container that runs the sample application.
+By default, the values from the binding data of the database service are projected as files into the workload container that runs the sample application. For example, all the values from the Secret resource are projected into the `bindings/spring-petclinic-pgcluster` directory.
 
-. To verify that the files in the application contain the projected binding data, use the following command in shell to print out the contents:
+. Verify that the files in the application contain the projected binding data, by printing out the directory contents:
 +
+[source,terminal]
 ----
-$ for i in username password host port type; do oc exec -it deploy/spring-petclinic-rest -n my-postgresql -- /bin/bash -c 'find /bindings/*/'$i' -exec echo -n {}:" " \; -exec cat {} \;'; echo; done
+$ for i in username password host port type; do oc exec -it deploy/spring-petclinic -n my-petclinic -- /bin/bash -c 'find /bindings/*/'$i' -exec echo -n {}:" " \; -exec cat {} \;'; echo; done
```

Comment:
@Srivaralakshmi If you want to keep this verification step in the quickstart, the command needs to be slightly modified:

```suggestion
$ for i in username password host port type; do oc exec -it deploy/spring-petclinic -n my-petclinic -- /bin/sh -c 'find /bindings/*/'$i' -exec echo -n {}:" " \; -exec cat {} \;'; echo; done
```

Reply:
@pmacik In that case, I am removing this step.